odml/rdf/query_creator.py

```diff
@@ -271,7 +274,7 @@ def get_query(self, q_str=None, q_parser=None):
             self.q_dict = q_parser.parse_query_string(q_str)
         self._prepare_query()
 
-        use_ns = {"odml": Namespace("https://g-node.org/odml-rdf#"), "rdf": RDF}
+        use_ns = {"odml": odmlns, "rdf": RDF}
```

Comment:
This is not the only usage of the namespace. Please change the other occurrences as well.